src/components/Template/Template.stories.tsx

```diff
@@ -91,3 +114,14 @@ export const RightSidebarTemplate = () => (
         </footer>
     </div>
 );
+
+RightSidebarTemplate.story = {
```

Comment:
I think there needs to be some work (https://storybook.js.org/docs/addons/using-addons/ ?) to make the addons appear?

Reply:
What do you mean? Beyond what Monica has already done? Seems to be working to me:

<img width="1439" alt="Screen Shot 2020-06-19 at 2 23 03 PM" src="https://user-images.githubusercontent.com/1720093/85168453-64f65780-b238-11ea-85cf-1ca16c54feca.png">

Reply:
This doesn't show up for me at all -- I've pulled this branch run npm install.  
I see that Monica's PR is merged - does it show up for you here as well?  https://nypl.github.io/nypl-design-system/storybook/storybook-static/react/?path=/story/accordion--accordion-with-checkboxes  

Is there a button I need to click?

Reply:
It won't show designs for stories unless the designs have been added. We haven't added designs for all of our stories. A better example would be [Button](https://nypl.github.io/nypl-design-system/storybook/storybook-static/react/?path=/story/button--button-filled-icon), where we have added designs. You should see a Figma embed when you click on the "Design" panel.

Reply:
You're not seeing the Design panel on `gh-pages` or locally?